package.json

```diff
@@ -11,9 +11,6 @@
     "truffle.js",
     "tsconfig.json"
   ],
-  "config": {
-    "gasLimit": "5600000"
```

Comment:
Why is this removed ?
pr #472 need that,

Reply:
restored